utilities/resources/spark_application.py

```
@@ -0,0 +1,312 @@
+        node_selector: dict[str, Any] | None = None,
+        proxy_user: str | None = None,
+        python_version: str | None = None,
+        restart_policy: dict[str, Any] | None = None,
+        retry_interval: int | None = None,
+        spark_conf: dict[str, Any] | None = None,
+        spark_config_map: str | None = None,
+        spark_ui_options: dict[str, Any] | None = None,
+        spark_version: str | None = None,
+        suspend: bool | None = None,
+        time_to_live_seconds: int | None = None,
+        type: str | None = None,
```

Ruff (0.15.17)

[error] Function argument `type` is shadowing a Python builtin

(A002)